### DISCUSSION TITLE
How can I read body as a Buffer?

### DISCUSSION THREAD

**Discussion by zero734kr** [nodejs/undici#1050] [Q&A] [answered]
Is there something like ``Response.buffer()`` from the fetch API here?

Thank you

**Comment by szmarczak** [comment, marked as answer]
In the fetch API there's no `buffer` but there's [`arrayBuffer`](https://developer.mozilla.org/en-US/docs/Web/API/Response/arrayBuffer).

**Reply by zero734kr** [reply]
I checked it again and it looks like you're right. ``Response.buffer()`` only has it in the [node-fetch documentation](https://www.npmjs.com/package/node-fetch#buffer)

**Reply by zero734kr** [reply]
I switched to ``arrayBuffer``, thanks for helping
